src/contents/gitpod-dashboard.ts

```diff
@@ -12,26 +8,13 @@ const isSiteGitpod = (): boolean => {
 }
 
 export const config: PlasmoCSConfig = {
-  matches: ["https://*/*"]
+  matches: [
+    "https://gitpod.io/*",
+    "https://*.gitpod.cloud/*",
+    "https://*.gitpod.dev/*"
+  ],
 }
 
-const storage = new Storage();
-
-const automaticallyUpdateEndpoint = async () => {
-  if (await storage.get<boolean>(STORAGE_AUTOMATICALLY_DETECT_GITPOD) === false) {
-    return;
-  }
-  
-  const currentUserSetEndpoint = await storage.get(STORAGE_KEY_ADDRESS);
-  if (!currentUserSetEndpoint || currentUserSetEndpoint === DEFAULT_GITPOD_ENDPOINT) {
-    const currentHost = window.location.host;
-    if (currentHost !== new URL(DEFAULT_GITPOD_ENDPOINT).host) {
-      console.log(`Gitpod extension: switching default endpoint to ${currentHost}.`)
-      await storage.set(STORAGE_KEY_ADDRESS, parseEndpoint(currentHost));
-    }
-  }
-}
 if (isSiteGitpod()) {
   sessionStorage.setItem("browser-extension-installed", "true");
-  automaticallyUpdateEndpoint();
```

Comment:
Perhaps it should, I think it makes sense after we've enabled the Dedicated wildcards. I removed it initially, because when we don't have access to the underlying sites, it does not make sense to automatically enable them.

Reply:
Integrated it in back again.